Analysis/Tasks/PWGDQ/v0selector.cxx

```diff
@@ -0,0 +1,230 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+//
+// Example analysis task to select clean V0 sample
+// ========================
+//
+// This code loops over a V0Data table and produces some
+// standard analysis output. It requires either
+// the lambdakzerofinder or the lambdakzeroproducer tasks
+// to have been executed in the workflow (before).
+//
+//    Comments, questions, complaints, suggestions?
+//    Please write to:
+//    daiki.sekihata@cern.ch
+//
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ASoAHelpers.h"
+#include "ReconstructionDataFormats/Track.h"
+#include "AnalysisCore/RecoDecay.h"
+#include "AnalysisCore/trackUtilities.h"
+#include "AnalysisDataModel/StrangenessTables.h"
+#include "AnalysisCore/TrackSelection.h"
+#include "AnalysisDataModel/TrackSelectionTables.h"
+#include "AnalysisDataModel/EventSelection.h"
+#include "AnalysisDataModel/Centrality.h"
+
+#include <cstdio>
+#include <TFile.h>
+#include <TH2F.h>
+#include <TProfile.h>
+#include <TLorentzVector.h>
+#include <Math/Vector4D.h>
+#include <TPDGCode.h>
+#include <TDatabasePDG.h>
+#include <cmath>
+#include <array>
+#include <cstdlib>
+#include "Framework/ASoAHelpers.h"
+#include "AnalysisDataModel/PID/PIDResponse.h"
```

Comment:
Please check the include statements and remove the ones which are not used

Reply:
thank you. done.

Reply:
done again.